timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -23,54 +23,63 @@
-    future values of a time series given its past), and quantiles of forecast distributions.
+    ``TimeSeriesPredictor`` provides probabilistic (distributional) multi-step-ahead forecasts for univariate time
+    series. The forecast includes both the mean (i.e., conditional expectation of future values given the past), as
+    well as the quantiles of the forecast distribution, indicating the range of possible future outcomes.
 
-    ``TimeSeriesPredictor`` models are learned "globally" from a collection of time series; i.e., a set of time
-    series model parameters are shared across all time series to be predicted, in contrast to
-    classical "local" approaches such as ARIMA.
+    ``TimeSeriesPredictor`` fits both "global" deep learning models that are shared across all time series (e.g.,
+    DeepAR, Temporal Fusion Transformer), as well as "local" statistical models that are fit to each individual time
+    series (e.g., ARIMA, ETS).
 
-    ``TimeSeriesPredictor`` fits a variety of neural network-based forecasting models as well as Bayesian models such
-    as Prophet. It expects input data sets and outputs predictions in the
+    ``TimeSeriesPredictor`` expects input data and makes predictions in the
     :class:`~autogluon.timeseries.TimeSeriesDataFrame` format.
 
+
     Parameters
     ----------
     target : str, default = "target"
-        Name of column that contains the target values to forecast (i.e., numeric observations of the
-        time series). This column must contain numeric values, and missing target values
-        should be in a pandas compatible format:
+        Name of column that contains the target values to forecast (i.e., numeric observations of the time series).
+        This column must contain numeric values, and missing target values should be in a pandas compatible format:
         https://pandas.pydata.org/pandas-docs/stable/user_guide/missing_data.html
-    eval_metric : str, default = None
+        # TODO: Do we even handle missing values? If so, should we make this more prominent?
+    prediction_length : int, default = 1
+        The forecast horizon, i.e., How many time steps into the future the models should be trained to predict.
+        For example, if time series contain daily observations, setting ``prediction_length = 3`` will train
+        models that predict up to 3 days into the future from the most recent observation.
+    eval_metric : str, default = "mean_wQuantileLoss"
         Metric by which predictions will be ultimately evaluated on future test data. AutoGluon tunes hyperparameters
         in order to improve this metric on validation data, and ranks models (on validation data) according to this
-        metric. Available options include: "MASE", "MAPE", "sMAPE", "mean_wQuantileLoss".
+        metric. Available options:
+
+        # TODO: Should we add MSE and RMSE here?
+        # TODO: mean_wQuantileLoss is not discussed in GluonTS docs
```

Comment:
we can put a brief explainer that this is the mean of normalized quantile losses for specified `quantile_levels`, and temporarily have a link to Amazon Forecast if you'd like: https://docs.aws.amazon.com/forecast/latest/dg/metrics.html#metrics-wQL .

As a side note, we should not have TODOs left in the docstrings and should move them to the code if we have any left.